scriptharness/actions.py

```diff
@@ -6,8 +6,8 @@
 
 Attributes:
   LOGGER_NAME (str): logging.Logger name to use
-  STRINGS (dict): strings for actions.  In the future these may be in a
-    function to allow for localization.
+  STRINGS (Dict[str, Dict[str, str]]): strings for actions.  In the future
```

Comment:
I preferred doing it this way because as you just said, it is a free form which is not clear for someone that hasn't seen the code at all. I think it is just more clear this way.

Nevertheless the docstrings are not inflexible at all, you could define a new type and use it there instead of the nested dicts. Another option is to define your own class and use that type (I did that for the LoggingDict) but since it was a free form I thought it was clearer this way.